toolboxes/cmr/cmr_file_and_directory_handling.cpp

```diff
@@ -199,7 +199,7 @@ namespace Gadgetron
     }
 
     template<typename T>
-    void load_items(const std::string& workingdirectory, const std::vector<std::string>& items_list, std::vector<hoNDObjectArray<T> >& items)
+    void load_items(const std::string& workingdirectory, const std::vector<std::string>& items_list, std::vector<hoNDArray<T> >& items)
```

Comment:
All of the stuff in these "cmr_file_and_directory_handling" stuff is not used (in the main Gadgetron). It is also not specifically CMR related. I don't have full visibility here, but this points in the direction of this being used downstream to store intermediate stuff. Few comments on this:

1. If it is just not being used right here but downstream projects depend on it, we should (at least) have some unit tests here that validate that these functions work and keep working. 
2. We would suggest using the storage server since this will not work well in distributed, etc. deployments. So I would prefer to see this removed all together, but again if we have downstream stuff that depends on it, we should consider that.

Reply:
This file is removed. Future should use storage server.